openfisca_core/reforms.py

```diff
@@ -126,58 +126,43 @@ def build_node(node, path_index):
 
 
 def update_items(items, start, stop, value):
-    def is_contained_in(inner, outer):
-        '''Returns True if `inner` is contained in `outer`.'''
-        return inner['start'] >= outer['start'] and (
-            outer.get('stop') is None or inner.get('stop') is not None and inner['stop'] <= outer['stop']
-            )
-    items = sorted(items, key=lambda item: item['start'])
-    new_item = create_item(start, stop, value)
-    split_items = list(split_item_containing_instant(start, items))
-    if stop is not None:
-        split_items = list(split_item_containing_instant(stop.offset(+1, 'day'), split_items))
-    not_contained_items = list(filter(lambda item: not is_contained_in(item, new_item), split_items))
-    if not_contained_items and not_contained_items[-1].get('stop') is None \
-            and str(start) > not_contained_items[-1]['start']:
-        last_not_contained_item = not_contained_items[-1].copy()  # Copy to avoid modifying `items` argument.
-        last_not_contained_item['stop'] = str(start.offset(-1, 'day'))
-        not_contained_items[-1] = last_not_contained_item
-    new_items = sorted(not_contained_items + [new_item], key=lambda item: item['start'])
-    return new_items
-
-
-def create_item(start, stop, value):
```

Comment:
If you are deprecating and removing this method, you must mention it in the changelog.